gremlin-driver/src/main/java/org/apache/tinkerpop/gremlin/driver/Connection.java

```diff
@@ -111,15 +112,15 @@ public Connection(final URI uri, final ConnectionPool pool) throws ConnectionExc
                 // close did not come from the client side. it means the server closed the channel for some reason.
                 // it's important to distinguish that difference in debugging
                 if (thisConnection.closeFuture.get() == null) {
-                    logger.error(String.format(
-                            "Server closed the Connection on channel %s - scheduling removal from %s",
+                    logger.info(String.format(
```

Comment:
Why is this changed from error to info? Because this is still used to capture cases where the server shut the connection, it should probably still be a higher value like "error"?

Reply:
Added check that if the connection was closed and it wasn't due to idle timeout, then to log this at error level.